deploy/lib/ingester.libsonnet

```diff
@@ -0,0 +1,99 @@
+local kube = import 'kube-libsonnet/kube.libsonnet';
+
+{
+    ingester_deployment:
+        local name = $._config.ingester.name;
+
+        # Arguments
+        local extraArgs = $._config.ingester.extraArgs;
+        local consul_uri = $._config.consul.name + '.' + $._config.namespace + '.svc.cluster.local:8500';
+        local args = [
+            '-ingester.join-after=30s',
+            '-ingester.claim-on-rollout=false',
```

Comment:
this should be true

Reply:
thanks for this! We've been running our cortex this way and it's been OK, but I agree that setting this to `true` would be better for production use.